styling/Phoenix.html

```diff
@@ -35,7 +35,23 @@
+--></div></div></div></div></header>
+
+
+<div class="container body-container"><div><div id="subheader-columns"><div class="row">
+<div class="col-xs-4 subheader"><h4>How is Phoenix different?</h4>
+<p>Phoenix is a framework for building HTML5 apps, API backends and distributed systems. Written in Elixir, you get a beautiful syntax, a productive tooling and fast runtime.
```

Comment:
Another idea, bold the most important parts for quick scanning `<strong>beautiful syntax, a productive tooling and fast runtime</strong>`